browser/ui/darker_theme/features.cc

```diff
@@ -8,7 +8,7 @@
 namespace darker_theme::features {
 
 BASE_FEATURE(kBraveDarkerTheme,
```

Comment:
why not replace `kBraveDarkerTheme` with `kBraveMidnightTheme` also?

Reply:
For one feature, we're saying it with two names - darker theme/midnight theme. do we have to this?
IMO, we can unify one - midnight theme.

Reply:
Or, why not just changing `about_flags.cc` if what we want is showing it as `midnight` to users?
I think it's sufficient. It's strange that its variable is darker and its name is midnight.

Reply:
Ah, I realized that this name could be used by users via command line switch.
Ok, it makes sense to have midnight here. It would be great to have some comments that it's darker in the code but expoed as midnight.